webapp/Connector/src/view/FilterStatus.js

```diff
@@ -68,22 +137,131 @@ Ext.define('Connector.view.FilterStatus', {
                 xtype: 'button', 
                 text: 'clear',
                 ui: 'rounded-small',
-                cls: 'filter-hdr-btn filterclear' /* for tests */,
+                cls: 'filter-hdr-btn filter-clear-btn' /* for tests */,
                 itemId: 'clear',
-                hidden: hidden
-            },{
-                xtype: 'button', 
-                text: 'save', 
-                ui: 'rounded-inverted-accent-small', 
-                cls: 'filter-hdr-btn filtersave' /* for tests */, 
-                itemId: 'savegroup', hidden: hidden
+                hidden: hidden,
+                handler: function() {
+                    me.hideGroupLabel();
+                    me.hideSaveAsGroupBtn();
+                    me.hideGroupSavePanel();
+                    me.resetFilterStatusBackGroundColor();
+                    me.hideEditGroupBtn();
+                }
+            }]
+        };
+    },
+
+    getSavedGroupName: function () {
+        return {
+            xtype: 'container',
+            itemId: 'savedgroupname-itemid',
+            id: 'savedgroupname-id',
+            ui: 'custom',
+            cls: 'savedgroup-label-container',
+            hidden: true,
+            layout: {
+                type: 'hbox'
+            },
+            items: [{
+                xtype: 'box',
+                cls: 'savedgroup-label',
+                id: 'savedgroup-label-id',
+                tpl: new Ext.XTemplate(
```

Comment:
This doesn't need to be a template instance, you can also pass back the string literal definition since everything here is a config (not an object instance).